src/index.md

```diff
@@ -9,8 +9,8 @@ research in the climate technology space. In particular we support individuals
 with a specific project in mind.
 
 We offer a <span class="amount">£1,000</span> no strings attached grant, paid
-directly to your bank account. Payment is made in three parts, half upon
-acceptance, and the remainder paid out in the accordance of meeting the
+directly to your bank account. Payment is made in three parts, forty percent
```

Comment:
haha; good catch; people would've been getting 110%

Reply:
hehe, not bad, not bad..